pom.xml

```diff
@@ -56,6 +72,11 @@
             <artifactId>activation</artifactId>
             <version>1.1.1</version>
         </dependency>
+        <dependency>
```

Comment:
Don't download all the jars of AWS, just S3 would be enough for image upload. In this case https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-s3/1.11.611 is required.